rts/Lua/LuaHandleSynced.cpp

```diff
@@ -1219,6 +1219,37 @@ bool CSyncedLuaHandle::AllowResourceTransfer(int oldTeam, int newTeam, const cha
 	return allow;
 }
 
+/*** Called when excess resources are added.
+ * Accumulates all excesses within a single gameframe.
+ *
```

Comment:
Should document what happens if you generate more excess while inside this event.